bindings/python/fdb/impl.py

```diff
@@ -51,11 +51,6 @@ def __init__(self, parent):
         self._parent = parent
 
 
-class _ClusterOptions(object):
-    def __init__(self, cluster):
-        self._parent = weakref.proxy(cluster)
```

Comment:
Hmm, I think the idea is that there are no cluster options, so this object wasn't really serving any purpose at any API version. However, we could still include an `options` member of the `Cluster` class just in case someone was accessing it and not using it. Perhaps it should just be set to `None` or maybe an empty object. Also, I don't think we actually need `_set_option` on `Cluster`.

Reply:
Yeah, it seems like a pretty unlikely that this would break anyone given that there were no cluster options before.